doc/source/programs/gdal_rasterize.rst

```diff
@@ -191,11 +191,12 @@ raster data is only supported since GDAL 2.1.0.
 
 .. option:: -optim {AUTO|VECTOR|RASTER}
 
-    Force the algorithm used (results are identical). The raster mode is used in most cases and
-    optimise read/write operations. The vector mode is useful with a decent amount of input
-    features and optimise the CPU use. That mode have to be used with tiled images to be
-    efficient. The auto mode (the default) will chose the algorithm based on input and output
-    properties.
+    Force the algorithm used (results are identical). Raster mode
+    is used in most cases and  optimizes  read/write  operations.  The
+    vector  mode is useful with a large amount of input features and
+    optimizes CPU use.  Tiled images also need to use vector mode
+    to be efficient. Auto mode (the default) will chose the
```

Comment:
I believe you changed the meaning wrongly. Vector mode can be efficient only when the output raster is tiled.  Tiled output raster does not require vector mode.
I had a feeling that "decent" means here "not a huge number of input features" but it maybe I am wrong.

Reply:
I assure you that I have a huge stack of coins to flip when trying to guess the meaning of these. So would be ever grateful if someone who knows better than me steps in and words it better, please. Thanks!